e2e/constants/selectors/commons.ts

```diff
@@ -0,0 +1,8 @@
+export const NAVIGATION_SELECTORS = {
```

Comment:
Make variable names as meaningful as possible.

```suggestion
export const NAVIGATION_BAR_SELECTORS = {
```